sections/02_intro.md

```diff
@@ -34,88 +34,118 @@ learning methods more challenging or less fruitful to apply.
 ### What is deep learning?
 
 One version:
-[This section needs citations]
+`TODO: this section needs citations`
 
-Deep Learning is a collection of new techniques that together have recently demonstrated 
-breakthrough gains over existing approaches in several fields. 
+Deep Learning is a collection of new techniques that together have recently
+demonstrated  breakthrough gains over existing approaches in several fields.
 
 Deep learning is built on a very old idea, neural networks, that was first
-proposed in 1943 [doi:10.1007/BF02478259] as a model for how biological
-brains proces information. Since then, interest in neural networks a computational
-models has waxed and waned several times. This history is interesting in its own right  [@doi:10.1103/RevModPhys.34.135, @doi:10.1103/RevModPhys.34.135],
-but in recent years, with the advances of Deep Learning, attention has shifted back.
-
-Several important advances make the current surge of work done in this area possible.
-
-First, several easy to use software packages (Tensorflow, Caffe, Theano) now enable a much broader range of scientists
-to build and train complicated. In the past, neural networks required very specialized knoweldge to 
-build and modify, including the ability to robustly code differentials of matrix 
-expressions. Errors here are often subtle and difficult to detect, so it could be 
-very difffult to tailor networks to specific problems without substantial experience
-and training. Now, however, with these new packages, even very complex neural networks 
-are automatically differentiated, and high level scripting instructions can transparently
-run very efficently on GPUs. The technology has progressed to the point that even
-algorithms can be differentiated [cite neural stack and neural memory papers].
-
-Second, key technical insight has been uncovered that guides the construction of 
```

Comment:
I'd suggest that this second point goes first. The ability to make more complicated networks has lead to the availability of packages. Historical and logical order.